### PR TITLE
Paul's edits to 0.3 upgrade guide

### PR DESCRIPTION
**Pull request by pdaoust**
some final formatting fixes and a rejiggering of the site structure and redirects. Apologies; the diff is really funky -- the formatting fixes are almost all whitespace/punctuation but it doesn't highlight the differences, and one file is shown as being deleted and recreated, not moved.

### CODE REVIEW

File: src/pages/resources/upgrade/upgrade-holochain-0.3.md [hunk 1]
```diff
@@ -223,24 +214,24 @@ This macro was renamed for consistency with other macros. The changelog entry fo
...
 
 ```rust
-#[derive(Serialize, Deserialize)]
-#[serde(tag = "type")]
-#[hdk_entry_types]
-#[unit_enum(UnitEntryTypes)]
+[derive(Serialize, Deserialize)]
+[serde(tag = "type")]
+[hdk_entry_types]
+[unit_enum(UnitEntryTypes)]
```
Comment (ThetaSinner) [inline src/pages/resources/upgrade/upgrade-holochain-0.3.md:232]:
```suggestion
#[derive(Serialize, Deserialize)]
#[serde(tag = "type")]
#[hdk_entry_types]
#[unit_enum(UnitEntryTypes)]
```

**Comment by ThetaSinner** [comment]
Currently not building due to a broken link?

File: src/pages/resources/upgrade/upgrade-holochain-0.3.md [hunk 2]
```diff
@@ -223,24 +214,24 @@ This macro was renamed for consistency with other macros. The changelog entry fo
...
 ```
 
 Which should be updated to:
 
 ```rust
-#[derive(Serialize, Deserialize)]
-#[serde(tag = "type")]
-#[hdk_entry_types]
-#[unit_enum(UnitEntryTypes)]
+[derive(Serialize, Deserialize)]
```
[the remark this answers is not in the document]
Comment (pdaoust) [reply src/pages/resources/upgrade/upgrade-holochain-0.3.md:229]: aha! sure was. `^#` was a little too wild of a pattern, I guess... wonder why it spared the three lines before

**Comment by pdaoust** [comment]
Broken link fixed; merging now!

**Pull request merged**